### ISSUE TITLE
how to set default sensor location for multiple house (improvement)

### ISSUE THREAD

**Issue by fildy** [lroguet/amzn-alexa-skill-netatmo#11]
i have several netatmo that installed in several house.
until now, it's not posible to select which one of sensor location will be reported by alexa

**Comment by lroguet** [comment]
Hi,

This would definitely be a nice feature to have but not very high on my TODO list I'm afraid.

**Comment by fildy** [comment]
I agree. Once the skill is more robust, I hope we can add this feature
